photos.graphql

```diff
@@ -36,23 +33,26 @@ type Query {
 }
 
 type User
-  @join__owner(graph: AUTH)
-  @join__type(graph: AUTH, key: "id")
-  @join__type(graph: ALBUMS, key: "id") {
-  id: ID @join__field(graph: AUTH)
+    @join__owner(graph: AUTH)
```

Comment:
None of the reasonable formatting options for this are great to look at IMO. But I think I'd prefer the curly on the next line over this indentation?

i.e.
```graphql
type Album
  @join__owner(graph: ALBUMS)
  @join__types(graph: ALBUMS, key: "id")
{
  id: ID!
  user: User
  photos: [Image!]
}
```

Reply:
cool da83222